app/controllers/comments_controller.rb

```diff
@@ -65,11 +65,11 @@ def find_comment
   end
 
   def set_comments_made
-    @comments_made = Comment.where(commenter_id: @user.id).public_or_private(display_private_comments?).page(params[:made_page])
+    @pagy_comments_made, @comments_made = pagy(Comment.where(commenter_id: @user.id).public_or_private(display_private_comments?))
```

Comment:
@innatewonderer Will paginate in this case was not using `params[:page]` but `params[:made_page]` (as in comments made) — the problem is that we have 2 things paginating on this page, so we don't want 1 param to control them both. Can you look into how to use different param names for this instance?

Reply:
Hmmm, I thought I was doing that by naming one thing `@pagy`, and another one `@pagy_comments_made`... but you are saying that the actual query doesn't return only `comments made` comments? interesting 🤔 Let me look into it.

Reply:
Got it. Fixed.